src/libs/actions/IOU.ts

```diff
@@ -1031,14 +1031,21 @@ function createDraftTransaction(transaction: OnyxTypes.Transaction) {
     Onyx.set(`${ONYXKEYS.COLLECTION.TRANSACTION_DRAFT}${transaction.transactionID}`, newTransaction);
 }
 
-function clearMoneyRequest(transactionID: string, skipConfirmation = false) {
-    removeDraftTransactions();
+function clearMoneyRequest(transactionID: string, skipConfirmation = false, draftsTransaction?: OnyxCollection<OnyxTypes.Transaction>) {
```

Comment:
This is referring to a single draft right? If so I think we should change the variable name everywhere in this file like this

```suggestion
function clearMoneyRequest(transactionID: string, skipConfirmation = false, draftTransaction?: OnyxCollection<OnyxTypes.Transaction>) {
```

If it _is_ supposed to be plural then it should be 
```suggestion
function clearMoneyRequest(transactionID: string, skipConfirmation = false, draftTransactions?: OnyxCollection<OnyxTypes.Transaction>) {
```